lib/dynflow/testing/in_thread_executor.rb

```diff
@@ -34,6 +34,13 @@ def event(execution_plan_id, step_id, event, future = Concurrent::Promises.resol
         future
       end
 
+      def delayed_event(director_event)
+        @director.handle_event(director_event).each do |work_item|
+          @work_items << work_item
+        end
```

Comment:
Isn't this basically `@work_items.concat @director.handle_event(director_event)`?

Reply:
:dart: I was copying over... will fix in both places :)

Reply:
It seems, nut not really... `@work_items` is a queue not an array and as such doesn't have concat :)

Reply:
Oh, my bad. Didn't realize it is not a regular Array